GestureHandler.js

```diff
@@ -450,6 +450,10 @@ class BaseButton extends React.Component {
     onActiveStateChange: PropTypes.func,
   };
 
+  static defaultProps = {
+    overflow: 'hidden',
```

Comment:
have you tested this? From what I remember overflow is a style property so it should go under `style` section. Can you also please verify that this setting can be overwritten (e.g. we could do `<RectButton style={{overflow: 'visible'}} />`)

Reply:
https://react-native.canny.io/feature-requests/p/style-defining-a-default-style-for-components
Default styles is not allowed currently ion RN and I believe it will be available soon.
Now it works but it's impossible to override `overflow`

Reply:
```
this { style, ...rest } = this.props;

<RawButton style={[{overflow: 'hidden' }, style]} {...rest} onGesture.... />
```